components/kserve/resources/servicemesh/kserve-inferencegraph-authorizationpolicy.tmpl.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: security.istio.io/v1beta1
```

Comment:
move it to https://github.com/opendatahub-io/opendatahub-operator/tree/main/controllers/components/kserve/resources/servicemesh